index.bs

```diff
@@ -11346,6 +11354,187 @@ The [=remote end steps=] given |session| and |command parameters| are:
 </div>
 
 
+## The webExtension Module ## {#module-webExtension}
+
+The <dfn export for=modules>webExtension</dfn> module contains functionality for
+managing and interacting with web extensions.
+
+### Definition ### {#module-webExtension-definition}
+
+[=remote end definition=]
+
+<pre class="cddl remote-cddl">
+WebExtensionsCommand = (
+  webExtension.Install,
+  webExtension.Uninstall
+)
+</pre>
+
+[=local end definition=]
+
+<pre class="cddl local-cddl">
+WebExtensionsResult = (
+  webExtension.InstallResult
+)
+</pre>
+
+### Types ### {#module-webExtension-types}
+
+#### The webExtension.Extension Type #### {#type-webExtension-Extension}
+
+<pre class="cddl remote-cddl local-cddl">
+webExtension.Extension = text
+</pre>
+
+The <code>webExtension.Extension</code> type represents a web extension id within a browser.
```

Comment:
@jgraham should we actually use `remote end` for all those cases instead of `browser` to be consistent?

Reply:
I think remote end is marginally better, but arguably only an actual browser can have an extension. I think honestly we're not 100% consistent already (e.g. with the `browser` module)